### ISSUE TITLE
Missing indentation after `wrapAttributes` rule is applied

### ISSUE THREAD

**Issue by bennokress** [nicklockwood/SwiftFormat#1231]
I enabled the `wrapAttributes` rule on my project today with the `--funcattribute prev-line` option. When running the `swiftformat` command it applied the rule as expected, however it did not indent the newly created wrapped line. In my opinion the warped line should indent to the same level as its attributes.
### Before applying `swiftformat`
```swift
    @discardableResult func setActivityIndicatorColor(_ color: UIColor) -> [UIColor] {
        environment(\.activityIndicatorColors, [color])
        return [color]
    }
```
### After applying `swiftformat`
```swift
    @discardableResult
func setActivityIndicatorColor(_ color: UIColor) -> [UIColor] {
        environment(\.activityIndicatorColors, [color])
        return [color]
    }
```
### Expected behavior
```swift
    @discardableResult
    func setActivityIndicatorColor(_ color: UIColor) -> [UIColor] {
        environment(\.activityIndicatorColors, [color])
        return [color]
    }
```

**Comment by nicklockwood** [comment]
@bennokress you're right, this looks like a bug. I missed it because normally the --indent rule would apply the correct indentation in the same pass.

**Comment by nicklockwood** [comment]
@bennokress fixed in 0.49.14
